tools/cross-compile.sh

```diff
@@ -1,120 +1,107 @@
-cd ..
-
-mkdir -p binaries
-mkdir -p micro-$1
-
-cp LICENSE micro-$1
-cp README.md micro-$1
-cp LICENSE-THIRD-PARTY micro-$1
-cp assets/packaging/micro.1 micro-$1
-cp assets/packaging/micro.desktop micro-$1
-cp assets/micro-logo-mark.svg micro-$1/micro.svg
+#!/bin/sh
```

Comment:
Maybe it's a good idea to use `#!/bin/sh -e`? Any of the commands in this script may potentially fails, so we'd better not ignore such failures silently?

What do these github actions do if the command (`cross-compile.sh` in this case) exits with a non-zero status? I hope they abort the action and log the stderr output?

Reply:
`set -e` added. The action will stop at the first error and not proceed the upcoming steps. As far as I've seen while testing in my fork it prints the error which appeared.